lisp/ess-custom.el

```diff
@@ -2108,27 +2113,26 @@ for help files.  The default value is nil for other systems."
 
 ;;;;; names for communication using MS-Windows 9x/NT ddeclient mechanism
 
-(defcustom inferior-ess-ddeclient nil
+(defcustom inferior-ess-ddeclient "Initial"
```

Comment:
> Why "Initial". Looks like a wart.

yes, someone initialised that variable to "Initial" in ess-site.el... I should have changed the code to deal with nil rather than "Initial" but I think I was starting to give up at that point ;) I'll change it to nil.

> I think we can drop support for DDE. I doubt anyone uses that anymore.

It's needed for S-plus on Windows. I'm not against dropping support, but let's see if @rmheiberger manages to run the latest ESS with these environments.

> No-one tests is anyways.

Yes, we really need basic tests for all the inferiors officially supported, and some people on the team in charge of running them before each release. I think we should drop support for all the systems we are not able to test.